04-spatial-operations.Rmd

```diff
@@ -22,17 +23,21 @@ Spatial operations are an important component of any geospatial software and vit
 There are clear overlaps between spatial and non-spatial operations.
 Common spatial attribute data processing tasks include spatial subsetting (as we will see in section \@ref(spatial-subsetting) and section \@ref(raster-subsetting)), joining and aggregation (sections \@ref(spatial-joining-and-aggregation), \@ref(map-algebra) & \ref(merging-rasters)).
 In the case of vector data, each of these spatial operations has a non-spatial equivalent, as demonstrated in section \@ref(vector-attribute-manipulation) in the previous chapter.
-Of course, spatial raster and vector operations share many similarities, at least in terms of functionality.
-The internal processing is completely different. 
-In contrast to vector operations, subsetting is the only spatial raster operation we present in this chapter (section \@ref(spatial-operations-on-raster-data)) which also has a non-spatial counterpart.
+By contrast, subsetting is the only spatial raster operation we present in this chapter (section \@ref(spatial-operations-on-raster-data)) which also has a non-spatial counterpart.
+Nevertheless, spatial raster and vector operations share many similarities, at least in terms of functionality though the internal processing is completely different.
+Be careful with the wording.
+Sometimes the same words have slightly different meanings for both data models.
+<!-- maybe we should choose dissolving as the term here, aggregate is more R lingo-->
```

Comment:
Well, one to think about. I guess, aggregate is more R lingo, in the GIS world you talk of dissolving. Or we just have to make clear that aggregate and dissolve means the same when it comes to vector data.

Reply:
I think dissolve is the spatial operation that accompanies the attribute aggregation. Definitely an improvement the way you've worded it - may reword it slightly.